app/models/qc_asset_creator.rb

```diff
@@ -15,17 +15,6 @@ def initialize(api:,asset:,user:,purpose:,sibling: nil,template: nil, tag2_tubes
     self.extend behaviour_module
   end
 
-  def tag2_locations_for_barcode(barcode)
```

Comment:
Moved these into the corresponding behaviour module. They are only used in one behaviour